clang/lib/Driver/Driver.cpp

```diff
@@ -6726,7 +6855,8 @@ class OffloadingActionBuilder final {
       // We expect that the host action was just appended to the action list
       // before this method was called.
       assert(HostAction == AL.back() && "Host action not in the list??");
-      HostAction = C.MakeAction<OffloadBundlingJobAction>(OffloadAL);
+      auto *BundlingAction = C.MakeAction<OffloadBundlingJobAction>(OffloadAL);
```

Comment:
Do we need this change? I might be missing it but I don't see it affecting anything

Reply:
Yeah - not needed.  I must have been working against `BundlingAction` in a previous set of changes that I didn't need anymore.